README.md

```diff
@@ -1,3 +1,135 @@
 # scats
-
 Selenium Cucumber automated testing system
+
```

Comment:
yes, @phuonglm will help with that. We have a maven repo too, so basically push snapshots to Teracy maven repo for testing, when we have the official release, @phuonglm will push it to Maven Central repo.